packages/@aws-cdk/aws-lambda-nodejs/lib/bundling.ts

```diff
@@ -140,6 +140,10 @@ export class Bundling implements cdk.BundlingOptions {
       ...loaders.map(([ext, name]) => `--loader:${ext}=${name}`),
       ...this.props.logLevel ? [`--log-level=${this.props.logLevel}`] : [],
       ...this.props.keepNames ? ['--keep-names'] : [],
+      ...this.props.tsConfig ? [`--tsconfig=${pathJoin(inputDir, this.props.tsConfig)}`] : [],
+      ...this.props.metaFile ? [`--metafile=${pathJoin(outputDir, 'index.meta.json')}`] : [],
+      ...this.props.banner ? [`--banner='/* ${this.props.banner} */'`] : [],
+      ...this.props.footer ? [`--footer='/* ${this.props.footer} */'`] : [],
```

Comment:
Why are we forcing this to be a comment. Technically users can also inject code like this. I'd just pass this through